### PR TITLE
GEOMESA-1637 Correct Extra Classpaths

### PR DESCRIPTION
**Pull request by jahhulbert-ccri**
Signed-off-by: Andrew Hulbert <ahulbert@ccri.com>

### CODE REVIEW

File: geomesa-accumulo/geomesa-accumulo-tools/bin/geomesa [hunk 1]
```diff
@@ -57,7 +57,7 @@ fi
 GEOMESA_CP="${GEOMESA_CONF_DIR}:$(findJars $GEOMESA_LIB):${ACCUMULO_CONF_DIR}:${HADOOP_CONF_DIR}"
 
 # Add user defined directories to the classpath
-if [[ -z "$GEOMESA_EXTRA_CLASSPATHS"  ]]; then
+if [[ ! -z ${GEOMESA_EXTRA_CLASSPATHS+x}  ]]; then
```
Comment (jnh5y) [inline geomesa-accumulo/geomesa-accumulo-tools/bin/geomesa]: @aheyne does this make sense to you?

Comment (aheyne) [inline geomesa-accumulo/geomesa-accumulo-tools/bin/geomesa]: I guess it works but I don't really understand why you wouldn't just do 
`if [[ -n "${GEOMESA_EXTRA_CLASSPATHS}" ]]; then`
which seems more readable.

Comment (jahhulbert-ccri) [inline geomesa-accumulo/geomesa-accumulo-tools/bin/geomesa]: yup. i can make it that. I just quickly added the ! to fix the logic error

**Comment by jahhulbert-ccri** [comment]
found more places that it was wrong and standardized them to all look the same

**Comment by jahhulbert-ccri** [comment]
if someone else wants to try here's what I'd do...

```
vim reference.conf   # put a converter named remote-foobar in this
jar cvf /tmp/foobar.jar reference.conf

hadoop fs -put somefile.csv /user/you/somefile.csv

GEOMESA_EXTRA_CLASSPATHS=/tmp/foobar.jar geomesa ingest .... hdfs://namenode:9000/user/you/somefile.csv

```

**Pull request merged**